cypress/support/index.js

```diff
@@ -0,0 +1,8 @@
+import './commands';
+
+// Ignore uncaught exception so tests doesn't stop mid run
+Cypress.on('uncaught:exception', () => false);
```

Comment:
Can we log the exception first? we could still ignore it afterwards

Reply:
It still logs it in the browser console. It's just that, by default, as soon as there's an uncaught exception the tests fails and sadly, there's currently one just before keycloak redirects us to the app after the login. I would prefer to fix the error but it's not the best timing and "it works" so..